packages/stryker/package.json

```diff
@@ -65,9 +65,12 @@
     "progress": "^1.1.8",
     "rimraf": "^2.6.1",
     "serialize-javascript": "^1.3.0",
-    "tslib": "^1.5.0"
+    "tslib": "^1.5.0",
+    "inquirer": "^3.0.6"
```

Comment:
I think inquirer belongs to https://github.com/stryker-mutator/stryker/pull/269 no?